azure-pipelines.yml

```diff
@@ -32,6 +32,7 @@ stages:
             imageName: 'ubuntu-16.04'
             TILEDB_AZURE: ON
             TILEDB_STATIC: OFF
+            TILEDB_TESTS_ENABLE_ARROW: ON
```

Comment:
I suspect this came from resolving a merge conflict. We recently removed this environment variable. Please remove this line and L46 and 93.

Reply:
Yes, from merge conflict resolution. All three lines removed.